src/domain/usecases/program-rules-processing/ProgramRuleValidationForEGASP.ts

```diff
@@ -362,10 +359,8 @@ export class ProgramRuleValidationForEGASP {
             enrollmentId: undefined,
             enrollmentStatus: undefined,
             status: event.status,
-            eventDate: event.eventDate,
-            occurredAt: event.eventDate,
-            trackedEntityInstanceId: teiId,
```

Comment:
Check if trackedEntityInstanceId it is necessary if so then add trackedEntityInstanceId using `const teiId = event.trackedEntity;`

Reply:
Got it! Also used it here like it was before